### PR TITLE
Updated dotnet docs

### PR DESCRIPTION
**Pull request by krigar1184**

### CODE REVIEW

File: getting-started/dev/net.md [hunk 1]
```diff
@@ -4,28 +4,28 @@
...
-```cpp
+```cs
 using Lucidtech.Las;
 
-ApiClient apiClient = new ApiClient("<endpoint>");
-Prediction response = apiClient.Predict(documentPath: "document.pdf", modelName: "invoice|receipt|documentSplit");
-Console.WriteLine(response.ToJsonString(Formatting.Indented));
+Client apiClient = new Client(<credentials>);
+string documentId = "las:document:39e2d47b04214610a66050d1c6ed8b6d";
+string modelId = "las:document:39e2d47b04214610a66050d1c6ed8b6d";
```
Comment (augustak) [inline getting-started/dev/net.md]: las:model:...

**Review by augustak — COMMENTED** [review]
Can you update SUMMARY.md with a latest reference to new docs? Similar to how it's done with Python

File: getting-started/dev/net.md [hunk 2]
```diff
@@ -4,28 +4,28 @@
...
 Suppose we wish to run inference on a document using Lucidtech’s invoice model.
 
-```cpp
+```cs
 using Lucidtech.Las;
 
-ApiClient apiClient = new ApiClient("<endpoint>");
-Prediction response = apiClient.Predict(documentPath: "document.pdf", modelName: "invoice|receipt|documentSplit");
-Console.WriteLine(response.ToJsonString(Formatting.Indented));
+Client apiClient = new Client(<credentials>);
```
Comment (augustak) [inline getting-started/dev/net.md]: Just 'client' now?

File: getting-started/dev/net.md [hunk 3]
```diff
@@ -4,28 +4,28 @@
...
 
-var feedback = new List<Dictionary<string, string>>()
+var groundTruth = new List<Dictionary<string, string>>()
 { 
     new Dictionary<string, string>(){{"label", "total_amount"},{"value", "54.50"}},
     new Dictionary<string, string>(){{"label", "purchase_date"},{"value", "2007-07-30"}}
 };
-FeedbackResponse response = apiClient.SendFeedback(documentId: "<documentId>", feedback: feedback);
-Console.WriteLine(response.ToJsonString(Formatting.Indented));
+var response = apiClient.SendFeedback(documentId: "<documentId>", groundTruth: groundTruth);
```
Comment (augustak) [inline getting-started/dev/net.md]: Should be client.UpdateDocument(...)

File: getting-started/dev/net.md [hunk 4]
```diff
@@ -34,10 +34,10 @@ Console.WriteLine(response.ToJsonString(Formatting.Indented));
 Consent ID is an identifier you can assign to documents to keep track of document ownership for your customers.
 {% endhint %}
 
-```cpp
+```cs
 using Lucidtech.Las;
 
-ApiClient apiClient = new ApiClient("<endpoint>");
+Client apiClient = new Client(<credentials>);
```
Comment (augustak) [inline getting-started/dev/net.md]: Just client

File: getting-started/dev/net.md [hunk 5]
```diff
@@ -46,11 +46,10 @@ var response = apiClient.CreateDocument(body, "application/pdf", "<consent id>")
...
 
-```cpp
+```cs
 using Lucidtech.Las;
 
-ApiClient apiClient = new ApiClient("<endpoint>");
-RevokeResponse response = apiClient.RevokeConsent(consentId: "<consentId>");
-Console.WriteLine(response.ToJsonString(Formatting.Indented));
+Client apiClient = new Client(<credentials>);
+var response = apiClient.DeleteDocuments(consentId: "<consentId>");
```
Comment (augustak) [inline getting-started/dev/net.md]: just client

**Review by augustak — CHANGES_REQUESTED** [review]

File: getting-started/dev/net.md (hunk 2 above)
Comment (krigar1184) [reply getting-started/dev/net.md]: If you mean variable name, I don't see the difference, but yes, `ApiClient` class has been removed.

**Review by augustak — APPROVED** [review]

**Pull request merged**